utils/python/CIME/utils.py

```diff
@@ -550,6 +543,14 @@ def handle_standard_logging_options(args):
     root_logger=logging.getLogger()
     # DEBUG trumps INFO trumps Silent (WARN)
     if (args.debug == True):
+        file1 = {
```

Comment:
This change fixes the problem of writing an empty scripts.log file when you run without --debug while still printing a complete debug log when the flag is set.